packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -42,21 +43,21 @@ class CodeView extends StatefulWidget {
 
 class _CodeViewState extends State<CodeView> with AutoDisposeMixin {
   Script script;
-  List<String> lines = [];
+  int lineCount = 0;
   Set<int> executableLines = {};
 
   LinkedScrollControllerGroup verticalController;
   ScrollController gutterController;
   ScrollController textController;
+  SyntaxHighlighter highlighter;
 
   ScriptRef get scriptRef => widget.scriptRef;
 
   @override
   void initState() {
     super.initState();
 
-    _initScriptInfo();
-
+    SyntaxHighlighter.initialize().then((_) => _initScriptInfo());
```

Comment:
It looks like the initialization happens asynchronously; I assume it wouldn't be ideal if the highlighter instance is created before the SyntaxHighlighter is inited. Perhaps move the async initialization elsewhere? Into some async debugger page initialization, or, into some other async initialization for the app>

Reply:
Good point. It looks like the only good place to do this is before we call `runApp`, so I've gone ahead and done that there.